### DISCUSSION TITLE
About NavneethJayendran

### DISCUSSION THREAD

**Discussion by ldez** [GaijinEntertainment/go-exhaustruct#77] [General]
Hello,

@xobotyi I'm sorry if I bother you and feel free to not answer.

There is a user of `exhaustruct` and `golangci-lint` that opens a discussion about your linter.

https://github.com/golangci/golangci-lint/discussions/4076

Can you clarify the situation of this user and maybe provide some context?

We can talk privately if you prefer.

**Comment by xobotyi** [comment]
I simply had  no spare time since the middle of july and just returned from long-awaited vacation🙃

About user that claims to be banned - i didnt do anything proactively to ban him, interaction limits are off and i cant see any other config that can prevent exact user from creating PRs🤔

**Reply by ldez** [reply]
A drama for nothing...

Thank you @xobotyi and sorry for the noise.

**Reply by navijation** [reply]
Thanks for getting back. I am able to interact at this point, mysteriously enough. I'll try to reach out more next time before trying to escalate.
